### DISCUSSION TITLE
Claw in SI Beta

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#275] [General]
Looking at the SI Beta, there is a cave with a staircase down on the southeast of the isle. That takes you back into the abandoned fort inside the ratmen's dungeon. So perhaps the route between Moonshade and Furnace was via Claw?

**Comment by KnightCaptainU7** [comment]
Makes a bit more sense with the heat from Furnace keeping the palm trees alive in a cooler world than Britannia.

**Comment by r-h-o-n-a** [comment]
Confirmed by the Moonshade plot document in multiple spots (see 8c and 8i, specifically).

Also along those lines, Part 8c says Fedabiblio is supposed to have a "half human & half cat" skeleton in his rooms. In the commercial release one of Fedabiblio's tables bears an anonymous journal detailing experiments on cats, clearly a reference to Claw (it also overtly mentions that the side effect of one of the experiments was "**that strange glow which surrounds them**", as with the glowing dead cat you're giving to Iolo in the teleport storm); the same table in Fedabiblio's rooms has a skull on it, which probably was meant at first to be the cat skull (244x2) but changed to a human skull when the Claw subplot was cut.

On a related note, in terms of additional NPCs who might join the party, have you considered Yurel? I always felt kind of bad that even after you solve Issik's maze he ends up just wandering around in there aimlessly, still saying he wants to go home. Having him join so that he could go back to Claw might be an interesting side quest.
